draft-ietf-cbor-7049bis.md

```diff
@@ -630,26 +654,15 @@ BF           -- Start indefinite-length map
 
 ### Indefinite-Length Byte Strings and Text Strings
 
-Indefinite-length byte strings and text strings are actually a
-concatenation of zero or more definite-length byte or text strings
-("chunks") that are together treated as one contiguous
-string. Indefinite-length strings are opened with the major type and
-additional information value of 31, but what follows are a series of
-byte or text strings that have definite lengths (the chunks). The end
-of the series of chunks is indicated by encoding the "break" stop code
-(0b111_11111) in a place where the next chunk in the series would
-occur. The contents of the chunks are concatenated together, and the
-overall length of the indefinite-length string will be the sum of the
-lengths of all of the chunks.  In summary, an indefinite-length string
-is encoded similarly to how an indefinite-length array of its chunks
-would be encoded, except that the major type of the indefinite-length
-string is that of a (text or byte) string and matches the major types
-of its chunks.
-
-For indefinite-length byte strings, every data item (chunk) between
-the indefinite-length indicator and the "break" MUST be a
-definite-length byte string item; if the parser sees any item type
-other than a byte string before it sees the "break", it is an error.
+Indefinite-length strings consist of a byte containing the major type
+and additional information value of 31, followed by a series of byte
+or text strings ("chunks") that have definite lengths, followed by the
+"break" stop code ({{break}}).  The data item represented by the
+indefinite-length string is the concatenation of the chunks.
+
+If any item between the indefinite-length byte string indicator
```

Comment:
Ditto for text strings - generalize

Reply:
Here's an attempt, but I think it actually makes things less clear. Let me know if you have a better idea, want me to keep this, or want me to go back to the two separate statements.

Reply:
If any item between the indefinite-length byte string indicator (0b010_11111) and the "break" is not definite-length and is not the same major type, then the string is ill-formed.

Reply:
I can live with this, thanks.